### DISCUSSION TITLE
Add support for other editors

### DISCUSSION THREAD

**Discussion by underdpt** [spatie/ray#328] [Ideas]
Hi,

It would be great to add support for other editors like VSCode,  Atom, etc., like [facade/ignition](https://github.com/facade/ignition) does ([reference](https://github.com/facade/ignition/blob/4d659cb9d36c2c5dc7b04c6e4e900febb73a151d/resources/js/components/Shared/editorUrl.js)). 

Thanks!

**Comment by freekmurze** [comment]
In the preferences you can already choose between different editors 👍

**Reply by underdpt** [reply]
Ok, sorry I missed that (i was looking for it on ray.php, not on the ray app itself!)
